sidebars.js

```diff
@@ -175,6 +161,14 @@ module.exports = {
         items: [
             'playground',
             'trophy-case',
+            'writing-rules/overview',
```

Comment:
I'd move playground and trophy-case to the bottom of the list. Just a suggestion though.

Reply:
Maybe Playground first and Trophy-case as the last? @s-santillan

Reply:
Ah... but there needs to be an explanation about writing rules first before we introduce Playground...